### PR TITLE
Emit TreasuryUpdated in Vault's Constructor

### PR DESCRIPTION
**Pull request by joaop21**
In the vault contract, the treasury always has a valid address, but this is not true in the subgraph. The treasury is always null in the subgraph until the `setTreasury` vault function is called and consequently emits the `TreasuryUpdated` event.
This PR emits the same event in the vault constructor to circumvent this problem.
Besides, this PR removes some useless docker deployment info that should not be in git.

### CODE REVIEW

File: test/Vault.spec.ts [hunk 1]
```diff
@@ -320,6 +320,12 @@ describe('Vault', () => {
       expect(await vault.treasury()).to.be.equal(TREASURY);
       expect(await vault.perfFeePct()).to.be.equal(PERFORMANCE_FEE_PCT);
     });
+
+    it('check the emission of the TreasuryUpdate event', async () => {
```
Comment (0xrin1) [inline test/Vault.spec.ts:324]: Pedantic but I would capitalize the `c` at the start of this test case description

**Review by 0xrin1 — APPROVED** [review]
LGTM

File: test/Vault.spec.ts (hunk 1 above)
Comment (brkomir) [reply test/Vault.spec.ts:324]: I would suggest changing this to `emits the TreasuryUpdated event`

**Review by brkomir — APPROVED** [review]

File: test/Vault.spec.ts (hunk 1 above)
Comment (0xrin1) [reply test/Vault.spec.ts]: @joaop21 fair point i just saw the test above and that one is capitalized nvm me :+1:

Comment (gabrielpoca) [reply test/Vault.spec.ts]: guys, tests should NOT start with a capitalize letter, the whole idea is to make it a readable line "it emits a TreasuryUpdated event", or "it checks the emission of the TreasuryUpdate event". But yeah, the first one is better

**Pull request merged**